README

```diff
@@ -41,32 +41,57 @@ The callback will only report on the status of tasks that have a name beinging w
 
 Note::
     If you haven't deployed using infrared, you can still run the tests if you create your own inventory file, containing one group of hosts called ``overcloud_nodes``.
-
+    Alternatively, you can create your own playbook that imports the tasks in ``tasks/*.yml``
 
 Adding new tests
 ----------------
 
-Adding a new test for existing deployment templates requires no changes in Jenkins.
-Adding tests for new deployment templates requires that the template is in Infrared and deployed in Jenkins.
+Adding a new test for existing deployment templates requires no changes in
+Jenkins.
+Adding tests for new deployment templates requires that the template is in
+Infrared and deployed in Jenkins.
+
+The changes required in this repo for tests is to add additional tasks under
+``tasks/test_<your_test_name>.yml``.
+
+```
+     - name: "[Test] My test name"
+       shell: |
```

Comment:
Just FYI you might want to suggest using `command` instead of `shell` when `bash` specific commands aren't required as that's the preferred (safer) way to execute things. I wouldn't hold this up though since the risk is pretty low I think.

Reply:
Opened an issue so I can circle back to this later

Reply:
Issue https://github.com/infrawatch/functional-tests/issues/7  is a reminder to check this